dom.bs

```diff
@@ -1217,7 +1217,32 @@ for discussion).
    <a>invoke</a> algorithm.
 
    <li><p>If <var>event</var>'s {{Event/type}} attribute value is not <var>listener</var>'s
-   <b>type</b>, terminate these substeps (and run them for the next <a>event listener</a>).
+   <b>type</b>, check for a legacy type attribute match by running these substeps:
```

Comment:
Just say "run these substeps". No need to have an explanation here, though you could have it in a note after the steps.